apps/src/templates/sectionProgressV2/AssignmentCompletionStatesBox.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import LegendItem from './LegendItem';
+import './section-progress-refresh.scss';
+import {StrongText} from '@cdo/apps/componentLibrary/typography';
+import {NOT_STARTED} from './IconKey';
+import color from '@cdo/apps/util/color';
+
+export default function AssignmentCompletionStatesBox({
+  isViewingLevelProgress,
+  hasValidatedLevels,
+}) {
+  // TO-DO (TEACH-800): Fix spacing on validated levels once width on page is set
+  const legendIcons = () => {
+    return (
+      <div className="icons-2">
+        <div className="legend-column-2">
```

Comment:
Are the classname `-2` just because they are in the second box? If so, could we name them more functionally. e.g. assignment-completion-column

Reply:
Done.